### PR TITLE
Lint examples folder

### PR DESCRIPTION
**Pull request by egonelbre**
Minimal changes to make linters pass

### CODE REVIEW

File: examples/eestream/store/main.go [hunk 1]
```diff
@@ -68,13 +68,20 @@ func Main() error {
 	errs := make(chan error, len(readers))
 	for i := range readers {
 		go func(i int) {
-			fh, err := os.Create(
-				filepath.Join(flag.Arg(0), fmt.Sprintf("%d.piece", i)))
+			piecefile := filepath.Join(flag.Arg(0), fmt.Sprintf("%d.piece", i))
```
Comment (kaloyan-raev) [inline examples/eestream/store/main.go]: This should be `pieceFile` to comply with Go naming convention.

File: examples/eestream/store/main.go [hunk 2]
```diff
@@ -68,13 +68,20 @@ func Main() error {
...
 				return
 			}
-			defer fh.Close()
+
+			defer func() {
+				err := fh.Close()
+				if err != nil {
+					fmt.Println(err)
+				}
+			}()
```
Comment (kaloyan-raev) [inline examples/eestream/store/main.go]: Can you do something about these deferred closes, so they don't look that ugly?

For the production code we have `utils.LogClose`, but it uses the zap logger to log any errors. Can we do anything similar for the examples too?

File: examples/statdb-client/main.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ import (
 
 var (
 	port   string
-	APIKey = []byte("abc123")
+	apikey = []byte("abc123")
```
Comment (kaloyan-raev) [inline examples/statdb-client/main.go]: This should be `apiKey` to comply with Go naming convention.

File: examples/eestream/store/main.go (hunk 2 above)
Comment (egonelbre) [reply examples/eestream/store/main.go]: I could make a func for each of the examples. I tried to make as little modifications as possible.

**Pull request merged**